monkey/infection_monkey/main.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import print_function
+
 
```

Comment:
Needless whitespace. 
Please run a linter over ALL the codebase so we can PEP8 it once and for all

Reply:
I would like to avoid making more py3 unrelated changes than is necessary on this PR. I'll fix whatever 2to3 or I left in terms of formatting, but I don't want to make this already big PR larger and more confusing.